dist/angular-validation.js

```diff
@@ -270,6 +270,28 @@
     };
 
     /**
+     * Add Message Element at the initial phase
+     * Do this function when to custom your messageElement 
+     * NODE: this funtion have similar purpose with `message-id` attribute. 
+     * This function will help you add your `messageElement` automatically instead of pre-defined.
+     * @param element
+     */
+    this.addMsgElement = function(element) {
+      return element.after('<span></span>');
```

Comment:
Sorry, I need to refresh everything.
The reason why I'm choosing `set` for the previous provider is they are allowed user to set something - set the html set the ouput message.

In this `addMsgElement`, this is correct when using within `directive`, because we are adding something not set. Set is the method that allow people to `set`, using `add` to do the set thing in config phase is not that correct (ideally)

@lvarayut @hungdoan2 Let's figure out a better way to use `set` or `add`